package/contents/ui/main.qml

```diff
@@ -7,53 +7,37 @@ import QtQuick 2.3
 import QtQuick.Layouts 1.0
 import QtQuick.Controls 2.0
 import org.kde.plasma.core 2.0 as PlasmaCore
-import org.kde.plasma.components 2.0 as PlasmaComponents
+import org.kde.plasma.components 3.0 as PlasmaComponents
 import org.kde.plasma.plasmoid 2.0
+import org.kde.plasma.extras 2.0 as PlasmaExtras
+import org.kde.kirigami 2.19 as Kirigami
 
 import QtWebEngine 1.9
-// import QtWebEngineCore 1.15
-// import QtWebChannel 1.0
 
 Item {
-	id:root
+	id: root
 	property bool themeMismatch: false;
 	property int nextReloadTime: 0
 	property int reloadRetries: 0
 
-	// Plasmoid.backgroundHints: plasmoid.configuration.showBackground ? PlasmaCore.Types.DefaultBackground : PlasmaCore.Types.NoBackground
-	Plasmoid.compactRepresentation: Item {
-		anchors.fill:parent
-		PlasmaCore.SvgItem {
-			anchors.centerIn : parent
-			width:parent.width < parent.height ? parent.width : parent.height
-			height:width
+	Plasmoid.compactRepresentation: CompactRepresentation {}
 
-			svg : PlasmaCore.Svg {
-				imagePath:Qt.resolvedUrl("assets/logo.svg");
-			}
-
-			MouseArea {
-				anchors.fill:parent
-				onClicked: {
-					plasmoid.expanded = !plasmoid.expanded
-				}
-			}
-		}
-	}
 	Plasmoid.fullRepresentation: ColumnLayout {
-		Layout.minimumWidth: 256 * PlasmaCore.Units.devicePixelRatio
-		Layout.minimumHeight:  512 * PlasmaCore.Units.devicePixelRatio
-		Layout.preferredWidth: 480 * PlasmaCore.Units.devicePixelRatio
-		Layout.preferredHeight: 920 * PlasmaCore.Units.devicePixelRatio
+		anchors.fill: parent
 
+		Layout.minimumWidth: 480 * PlasmaCore.Units.devicePixelRatio
+		Layout.minimumHeight:  720 * PlasmaCore.Units.devicePixelRatio
```

Comment:
Minimum   height/width  change  will be  reverted  as  it  was  asked  specifically  in the  past i will keep  the new preferred height/width